tools/add_triage_signature.py

```diff
@@ -396,19 +476,24 @@ def _update_ticket(self, url, issue_key, should_update=False):
             disks = inventory['disks']
             disks_details = defaultdict(list)
             for d in disks:
-                disks_details['type'].append(d.get('drive_type', ""))
+                disk_type = d.get('drive_type', "")
+                disks_details['type'].append(disk_type)
                 disks_details['bootable'].append(str(d.get('bootable', "NA")))
                 disks_details['name'].append(d.get('name', ""))
                 disks_details['path'].append(d.get('path', ""))
                 disks_details['by-path'].append(d.get('by_path', ""))
+                disks_details['smart'].append((self._parse_smart(d.get('smart', ""))
+                                               if disk_type != 'ODD' else
+                                               "{color:green}*N/A for optical drives*{color}"))
             hosts.append(OrderedDict(
                 id=host['id'],
                 hostname=self._get_hostname(host),
                 disk_name="\n".join(disks_details['name']),
                 disk_type="\n".join(disks_details['type']),
                 disk_path="\n".join(disks_details['path']),
                 disk_bootable="\n".join(disks_details['bootable']),
-                disk_by_path="\n".join(disks_details['by-path'])
+                disk_by_path="\n".join(disks_details['by-path']),
+                SMART="\n".join(disks_details['smart']),
```

Comment:
even though it's an acronym, I think it should stay ``smart`` because of Python conventions

Reply:
This kwarg is directly used as the header of the table shown to the user, and just lowercase "smart" is confusing - not everybody knows this acronym

Reply:
So change to ``"SMART": "value"``` (other values as well. Although it's weird we show ``disk_name`` and not ``Disk Name`` in a human readable table)

Reply:
Not sure I understand

Reply:
```suggestion
                "Disk Path": "\n".join(disks_details['path']),
                ...
                "SMART": "\n".join(disks_details['smart']),
```